changelog/db.changelog-1.3.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+ <changeSet author="Simon Dunn" id="1">
+    <tagDatabase tag="v1.2.0" />
+  </changeSet>
+  <changeSet author="Simon Dunn" id="2">
+    <addColumn tableName="settlements">
+      <column name="value" afterColumn="sourceSystem" type="INTEGER" />
+    </addColumn>
+  </changeSet>
+</databaseChangeLog>
```

Comment:
Another, another thing we want to do is **drop** the `invoiceNumber column from this table

We dnt save it here but more reference it via the `paymentRequests` table

Reply:
i think this will mean a rewrite to part of this ticket as i currently use the invoiceNumber to find duplicate settlements.
`app/inbound/get-settlement-by-invoicenumber-and-value.js`
What would be an alternative to using invoiceNumber to complete this functionality?

Reply:
When we save down a `settlement` we link the `paymentRequestId` for the record which matches it via the `invoiceNumber` in `paymentRequests`.

When we want to check for duplicates we can use this same approach of `invoiceNumber` from `paymentRequests` to get all the `paymentRequestId` which we can then only get the ones with the same `value` and you have -- just requires a call out to the `paymentRequests` table.

Reply:
so as discussed Friday i am going to leave `invoiceNumber` in the `settlements` table